### PR TITLE
Improve messaging around java password format

### PR DESCRIPTION
**Pull request by esoergel**
https://manage.dimagi.com/default.asp?257195
@dannyroberts

### CODE REVIEW

**Review by orangejenny — APPROVED** [review]
You tested locally, right? I've been bitten before by re-ordering things in commcare-app-settings that turn out to have depended on that ordering.

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -4948,8 +4948,12 @@ def check_password_charset(self):
         errors = []
         if hasattr(self, 'profile'):
             password_format = self.profile.get('properties', {}).get('password_format', 'n')
-            message = ('Your app requires {0} passwords '
-                       'but the admin password is not {0}')
+            message = (
```
Comment (orangejenny) [inline corehq/apps/app_manager/models.py]: trans?

**Comment by esoergel** [comment]
> You tested locally, right? I've been bitten before by re-ordering things in commcare-app-settings that turn out to have depended on that ordering.
I did test that the page loads and saves correctly, although I didn't deploy a J2ME app and test that out.

**Comment by orangejenny** [comment]
Wouldn't worry about testing end-to-end, the issue I'm thinking of happens on the HQ side.

File: corehq/apps/app_manager/models.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/app_manager/models.py]: updated

**Review by orangejenny — APPROVED** [review]

**Pull request merged**